examples/keyfiguresfinland.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+import pandas as pd
+from dash import Dash, dcc, html, Input, Output
+import dash_bootstrap_components as dbc
+import plotly.express as px
+import json
+
+# Data provided by Statistics Finland.
+# Original data source: https://pxdata.stat.fi/PxWeb/pxweb/en/Kuntien_avainluvut/Kuntien_avainluvut__2021/kuntien_avainluvut_2021_viimeisin.px/
+df = pd.read_csv('assets/key_figures_regions.csv', encoding = 'latin-1').rename(columns ={'Region 2021':'Region'}).set_index('Region')
+
+# Whole country figures on a pandas Dataframe.
+# Extract information from the key figure names to be used in callbacks.
+whole_country_df = pd.DataFrame(df.loc['WHOLE COUNTRY'])
+whole_country_df = whole_country_df.rename(columns = {'WHOLE COUNTRY':'value'})
+whole_country_df['year'] = [stat.split(', ')[-1] for stat in whole_country_df.index]
+whole_country_df['unit'] = [stat.split(', ')[-2] if len(stat.split(', ')) > 2 else '' for stat in whole_country_df.index]
+whole_country_df['stat_name'] = [stat.split(', ')[0] if len(stat.split(', ')) < 4 else ', '.join(stat.split(', ')[:2]) for stat in whole_country_df.index]
+
+# Drop since not needed.
+df.drop('WHOLE COUNTRY', axis = 0, inplace = True)
+
+# The json file used for the mapbox viz.
+# https://geo.stat.fi/geoserver/wfs?service=WFS&version=2.0.0&request=GetFeature&typeName=maakunta1000k_2021&outputFormat=json
+with open('assets/regions.json', encoding = 'utf-8') as f:
+    regions_json = json.loads(f.read())
+
+external_stylesheets = [dbc.themes.SPACELAB]
+
+app = Dash(name = __name__, external_stylesheets = external_stylesheets)
+
+app.layout = dbc.Container([
+        
+        html.Div("Finland's Regional Key Figures", className="text-center mb-3 mt-3 fw-bold display-1"),
+
+        dbc.Row([
+            
+            dbc.Col([
+                html.H2('Key figure'),
+                dcc.Dropdown(id = 'key-figures-finland-key-figure-selection-x',
+                             options = sorted(list(pd.unique(df.columns))),
+                             value = "Degree of urbanisation, %, 2020",
+                             className = 'text-dark bg-light text-nowrap'
+                             ),                
+                html.H1(id = 'key-figures-finland-whole-country-header-x', className="mt-5 display-2 text-center"),
+                html.Div(['Data by ',html.A('Statistics Finland', href = 'https://pxdata.stat.fi/PxWeb/pxweb/en/Kuntien_avainluvut/Kuntien_avainluvut__2021/kuntien_avainluvut_2021_viimeisin.px/', target = '_blank')], className="text-center fs-3 text"),
+                
+                ], xs = 12, sm = 12, md = 12, lg = 6, xl = 6, xxl = 6, align = 'center'),
```

Comment:
It would look better in the example index to have md=6.  Note this can be simplified:
from 
```
xs = 12, sm = 12, md = 12, lg = 6, xl = 6, xxl = 6
```

to:
```
md=6
```
This is because it applies to all screen sizes at the breakpoint and larger.  And the default is 12 so you don't have to set the smaller ones